engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -361,6 +362,18 @@ bool BaseRenderOpenGL3D::drawSprite(BaseSurfaceOpenGL3D &tex, const Wintermute::
 	return drawSpriteEx(tex, rect, pos, Vector2(0.0f, 0.0f), scale, 0.0f, color, alphaDisable, blendMode, mirrorX, mirrorY);
 }
 
+struct SpriteVertex {
```

Comment:
Since the usage of the struct depends on the structs byte-layout, it should be declared with the PACKED_STRUCT attribute:

struct X {
...
} PACKED_STRUCT;

Reply:
Done